javascript/javascript1/week1/EasyNames.js

```diff
@@ -0,0 +1,39 @@
+const firstWords = [
+  "Easy",
+  "Awesome",
+  "Fantastic",
+  "Great",
+  "Magical",
+  "Nifty",
+  "Simple",
+  "Jazzy",
+  "Helpful",
+  "Tremendous",
+];
+const secondWords = [
+  "Clams",
+  "Pies",
+  "Bees",
+  "Fire",
+  "Birds",
+  "Time",
+  "Puzzles",
+  "Project",
+  "Bananas",
+  "Company",
+];
+
+const randomNumber = Math.floor(Math.random() * 10);
+
+const startupName = firstWords[randomNumber] + " " + secondWords[randomNumber];
+const startupNameLength = startupName.length;
+
+console.log(
+  "The Startup: " +
+    "'" +
+    startupName +
+    "'" +
+    " contains " +
+    startupNameLength +
+    " characters."
+);
```

Comment:
Great job! 🔥

Your code is clean and well structured. 👌

My startupName became "Simple Puzzles" - now I just need to make a suitable product 😅🙌

Lastly, good attention to detail. Your string concatenation is on point. 👌 As you saw in FreeCodeCamp there are many ways to play around with inner quotes in a string, you can also use a forward slash as in the example below. 🤓

"this is a string with an inner quote \" place inner quote here \"."